src/services/content.ts

```diff
@@ -0,0 +1,10 @@
+import i18n from 'services/i18n'
+import contentManifest from 'content/manifest.json'
+
+export const getManifestFilesFromClasses = classes =>
```

Comment:
Is classes a string or a list of strings?

Reply:
list of strings

Reply:
A little `: string[]` would've been nice to clear that up 😇